fire/cli/niv/__init__.py

```diff
@@ -382,6 +385,21 @@ def punkt_feature(punkter: pd.DataFrame) -> Dict[str, str]:
         yield feature
 
 
+def bekræft2(spørgsmål: str) -> bool:
+    """
+    Bed bruger om at tage stilling til spørgsmålet.
+
+    Erstatning for bekræft().
```

Comment:
hvis den erstatter, så skal `bekræft()` vel elimineres - i alle `ilæg_*`-funktioner, hvorefter `bekræft2()` bliver til `bekræft()`. Det er lettere og sikkert også mere gennemskueligt hvis det gøres i et opfølgende PR., frem for at trække det ud i et præ-PR. Så måske enten eliminere teksten "Erstatning for bekræft()", eller ret den til "Kandidat til erstatning for bekræft()"

Reply:
Ja, `bekræft() skal elimineres og `bekræft2()` omdøbes. Tanken var umiddelbart at lave et løbende skift i takt med at jeg kommer omkring de enkelte ilæg-funktioner men det kan også tages i et hug en gang for alle.

Jeg fjerner den afsluttende linje, den var der mest for at formidle intentionen med funktionen til dig.